src/lib/core/style/focus-origin-monitor.ts

```diff
@@ -129,7 +129,7 @@ export class FocusOriginMonitor {
    * @param renderer The renderer to use to invoke the focus method on the element.
    * @param origin The focus origin.
    */
-  focusVia(element: HTMLElement, renderer: Renderer2, origin: FocusOrigin): void {
+  focusVia(element: HTMLElement, origin: FocusOrigin): void {
```

Comment:
This one is a breaking change that will need Google to be updated, so we should do it in a separate PR

(I checked, there are people using it)